src/routes/index.es

```diff
@@ -15,7 +15,7 @@ export const validations = {
 
 export const getParamReplacer = (typesValidation) => memoize((fragment, param, type) => {
   const regex = typesValidation[type];
-  return regex ? `${param}(${regex})` : fragment;
+  return regex ? `${param}(${regex})` : param;
```

Comment:
actually i think your original solution was correct. sorry.

this will break cases where you really need `<stuff>` in the url simply as is